src/Task/Git/CommitMessage.php

```diff
@@ -294,9 +294,14 @@ private function subjectIsSingleLined(ContextInterface $context): bool
     private function getCommitMessageLinesWithoutComments(string $commitMessage): array
     {
         $lines = preg_split('/\R/u', $commitMessage);
+        $everything_below_will_be_ignored = false;
```

Comment:
Can you use Camel Case ?

Reply:
Sure, committed the style change f732427